lib/path/default.nix

```diff
@@ -336,6 +336,34 @@ in /* No rec! Add dependencies on this file at the top. */ {
               ${subpathInvalidReason path}''
       ) 0 subpaths;
 
+  /* Split a subpath into its path component strings.
```

Comment:
Here _subpath_ should also be a link to a definition, but I'll merge this so you can do it in the other PR where I said the same.

Reply:
(for reference: https://github.com/NixOS/nixpkgs/pull/244358#discussion_r1268266262)

Let's conclude that thread first before continuing here.

Reply:
I now added a link to `subpath.isValid`, which after the above PR is merged will contain a definition for it. This PR is not dependent on that PR however.